File: official/docs/java/current/billing/create-stripe-credit-card-secret.java
package billing;

import com.easypost.exception.EasyPostException;
import com.easypost.model.PaymentMethod;
import com.easypost.service.EasyPostClient;

public class CreateStripeCreditCardSecret {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient("EASYPOST_API_KEY");

        ClientSecret response = client.betaReferralCustomer.createCreditCardClientSecret();

        System.out.println(response);
    }
}

File: official/docs/golang/current/billing/create-stripe-credit-card-secret.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v5"
)

func createStripeCreditCardSecret() {
	client := easypost.New("EASYPOST_API_KEY")

	response, _ := client.BetaCreateCreditCardClientSecret()

	fmt.Println(response)
}
